medallion/test/base_test.py

```diff
@@ -89,10 +89,9 @@ def setUp(self, start_threads=True):
         self.app_context = APPLICATION_INSTANCE.app_context()
         self.app_context.push()
         self.app.testing = True
-        if(not self.app.blueprints):
+        if not self.app.blueprints:
             register_blueprints(self.app)
         if self.type == "mongo":
-            reset_db(self.mongodb_config["backend"]["uri"])
```

Comment:
If we don't call `_clear_db()` in the mongo backend init function, call `_clear_db()` here.

Reply:
I can't the way it is currently coded - the connection to the db doesn't happen until line 119